app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -44,12 +44,12 @@ def instances
       instance["volumeIDs"].each do |vol_id|
         persister.disks.build(
```

Comment:
@agrare The disks we're populating here are the same block storage volumes added to our StorageManager. Is this for some other type of disk, or is it meant as a quick informational reference?

Reply:
Yeah we should be linking up the VM to the cloud_volumes rather than creating new disk records

Reply:
`collector.volume(vol_id)["size"]&.gigabytes`
One question. Here 'size' would evaluate to 'nil' if  "collector.volume(vol_id)["size"]" has the "nil" value, right? Will MIQ handle that?

Reply:
Yes, generally we treat `nil` as not available and only limited cases absolutely require the value to be present.  Most of those cases have model-level constraints validating the presence of the column.